crates/bevy_ecs/src/world/unsafe_world_cell.rs

```diff
@@ -157,6 +157,13 @@ impl<'w> UnsafeWorldCell<'w> {
         unsafe { &*self.0 }
     }
 
+    /// Retrieves this world's unique [ID](WorldId).
+    pub fn id(self) -> WorldId {
```

Comment:
```suggestion
    #[inline]
    pub fn id(self) -> WorldId {
```

The functions on `UnsafeWorldCell` has historically not inlined well.